Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -4089,6 +4090,27 @@ namespace AssetProcessor
                         newJob.m_scanFolder = scanFolder;
                         newJob.m_checkServer = jobDescriptor.m_checkServer;
 
+                        auto* uuidInterface = AZ::Interface<AssetProcessor::IUuidRequests>::Get();
+
+                        AZ_Assert(uuidInterface, "Programmer Error - IUuidRequests interface is not available.");
```

Comment:
Earlier in this PR in UuidUtilComponent::CreateSourceUuid, when an interface wasn't available, you checked for null and returned, here you aren't. I think checking for null + returning is usually safer than just crashing.

Reply:
Updated